src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+@import url(
+  "https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+);
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
<img width="1440" alt="Screenshot 2022-06-30 at 14 38 59" src="https://user-images.githubusercontent.com/94289038/176668569-b712c78d-faff-40fb-a4a0-2c840d8103c0.png">
box-shadow is absent